packages/mui-codemod/src/v5.0.0/jss-to-styled.test/multipleWithStyles.actual.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import withStyles from '@mui/styles/withStyles';
+
+const Button1 = withStyles({
+  root: {
+    backgroundColor: 'red',
+  },
+})(Button);
+
+const Button2 = withStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.primary.main,
+  },
+  actions: {
```

Comment:
Should we add a duplicate to a non `root` slot too? Are those going to have the suffix too?

Reply:
Sure, I will add to the test file.

Reply:
added.